scripts/plugins/extauth-hook-AD.py

```diff
@@ -27,6 +27,7 @@
 # - /etc/pam.d/hcp_users
 # - /etc/ssh/ssh_config
 
+#pylint: disable=super-with-arguments
```

Comment:
I'll be happy to review a PR that introduces pylint checks to the CI ;)

We should replace nosetests with pytest as well

Reply:
yes, there are several pylint warnings I should fix, 
but I like to leave it to anther commit to get your attention on the fix here.